.github/workflows/clang-tidy-review.yml

```diff
@@ -1,54 +1,44 @@
 name: clang-tidy-review
 
-on: [pull_request]
+on:
+  pull_request:
+    paths:
+      - '**.cxx'
+      - '**.hxx'
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
```

Comment:
Does it make sense to run the tests twice for push+PR? 
Or is that already sufficient to prevent this?

Reply:
Is there a way to check whether that changes anything? For most PRs they are based on the base branch, this is doing the same runs twice - if we could skip that, that would safe a lot of CPU time ...
But unless it is easy (which I worry it isn't) probably not worth to pursue ...

Reply:
That's a really good question! I guess if there are commits to the base branch that are more recent than any commits in the current branch, then it's worth doing the PR jobs? It's probably worth looking to see if there's something already existing that could handle that for us

Reply:
It looks to me like this is what we want:
https://github.com/fkirc/skip-duplicate-actions#example-1-skip-entire-jobs